lib/client-api/src/story_store.ts

```diff
@@ -503,33 +503,37 @@ export default class StoryStore {
       .map((i) => this.mergeAdditionalDataToStory(i));
   }
 
-  sortedStories(): StoreItem[] {
+  sortedStories(options: { normalizeParameters?: boolean } = {}): StoreItem[] {
+    // We need to pass the stories with denormalized parameters to the sort function (see #11010)
+    const denormalizedStories = mapValues(this._stories, (story) => ({
```

Comment:
Don't do the work if normalizeParameters is passed?

```js
    const stories = normalizeParameters ? this._stories : mapValues(this._stories, (story) => ({ ...
```

Reply:
We need to pass the denormalised stories to the sort function no matter what

Reply:
💡